pkg/storage/storage.go

```diff
@@ -31,8 +31,9 @@ type Driver interface {
 	StorageExists(*opapi.ImageRegistry, *bool) (bool, error)
 	RemoveStorage(*opapi.ImageRegistry, *bool) error
 	StorageChanged(*opapi.ImageRegistry, *bool) bool
-	GetStorageName(*opapi.ImageRegistry, *bool) (string, error)
+	GetStorageName() string
 	SyncSecrets(*coreapi.Secret) (map[string]string, error)
+	UpdateFromStorage(cfg opapi.ImageRegistryConfigStorage)
```

Comment:
Oh, we have so many methods in this interface. We need to document them.

Reply:
They are documented in s3.go, but that can be moved here in the future